packages/ia-topnav/src/ia-topnav.js

```diff
@@ -0,0 +1,205 @@
+import { LitElement, html } from 'lit-element';
+
+import './primary-nav';
+import './user-menu';
+import './search-menu';
+import './media-slider';
+import './desktop-subnav';
+import './dropdown-menu';
+import { user as userMenu } from './data/menus';
+import iaTopNavCSS from './styles/ia-topnav';
+
+export default class IATopNav extends LitElement {
+  static get styles() {
+    return iaTopNavCSS;
+  }
+
+  static get properties() {
+    return {
+      config: { type: Object },
+      mediaMenuOpen: { type: Boolean },
+      mediaSliderAnimate: { type: Boolean },
+      mediaSliderOpen: { type: Boolean },
+      searchIn: { type: String },
+      searchMenuAnimate: { type: Boolean },
+      searchMenuOpen: { type: Boolean },
+      selectedMenuOption: { type: String },
+      signedOutMenuOpen: { type: Boolean },
+      userMenuAnimate: { type: Boolean },
+      userMenuOpen: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.config = {};
+    this.mediaMenuOpen = false;
+    this.searchIn = '';
+    this.searchMenuAnimate = false;
+    this.searchMenuOpen = false;
+    this.signedOutMenuOpen = false;
+    this.userMenuAnimate = false;
+    this.userMenuOpen = false;
+    this.mediaSliderOpen = false;
```

Comment:
_Suggestion_ I think all of these `*Open` and `*Animate` properties would work better as methods on the target classes ie `userMenu.open()` and `userMenu.close()`. This class is having to keep a lot of these properties in sync whenever changes occur.

To keep track of the one that's currently open, you could use a single property like `currentlyOpenMenu` and whenever the menu changes you could update that one property.